flink-tests/src/test/java/org/apache/flink/test/accumulators/AccumulatorLiveITCase.java

```diff
@@ -142,18 +142,16 @@ public void testStreaming() throws Exception {
         submitJobAndVerifyResults(jobGraph);
     }
 
-    private static void submitJobAndVerifyResults(JobGraph jobGraph) throws Exception {
+    private void submitJobAndVerifyResults(JobGraph jobGraph) throws Exception {
```

Comment:
?

Reply:
Here is to use `getClass().getClassLoader()`, perhaps it is more appropriate to use `AccumulatorLiveITCase.class.getClassLoader()` or `CheckedThread.class.getClassLoader()` directly